doc/source/raysgd/v2/user_guide.rst

```diff
@@ -280,11 +526,11 @@ A couple caveats:
     analysis = tune.run(trainable, config={
         "lr": tune.uniform(), "batch_size": tune.randint(1, 2, 3)}, num_samples=12)
 
-
 Distributed metrics (for Pytorch)
 ---------------------------------
 
-In real applications, you may want to calcluate optimization metrics besides accuracy and loss: recall, precision, Fbeta, etc.
+In real applications, you may want to calculate optimization metrics besides
```

Comment:
These section feels out of place here, I think it should belong in the logging section.

Reply:
Yep makes sense, moved!